### PR TITLE
MW-543: Implemented adjustments to make sure that the command is

### PR DESCRIPTION
**Pull request by lajosarpad**
Adjusted the CLI command so it's more compatible with the message queue.

How to test:

- make sure you have a file, I'll call it `<yourtemplate>.lss`
- copy this file to `tmp/upload`, relative to the root folder
- run a proper CLI (see below)
- check the grid

Test command example without parameters

```
php application/commands/console.php importsurvey import-file tmp/upload/<yourtemplate>.lss
```

Test command example with parameters

```
php application/commands/console.php importsurvey import-file tmp/upload/<yourtemplate>.lss '{"bTranslateLinkFields": true, "sNewSurveyName": "Orsson", "DestSurveyID": 666999}'
```

Example after two imports

image

### CODE REVIEW

File: application/commands/ImportSurveyCommand.php [hunk 1]
```diff
@@ -46,14 +54,25 @@ public function run($sArgument)
...
+                if (!preg_match('/^[a-zA-Z0-9_\. ]*$/', $filename)) {
                     throw new Exception("Your filename can only contain letters, digits and dot");
                 }
                 $furtherParams = $sArgument[2] ?? null;
-                $this->importFile($filename, $furtherParams);
+                $result = $this->importFile($path, $filename, $furtherParams);
+                if (is_array($result) && isset($result['newsid'])) {
+                    echo $result['newsid'];
+                } else {
+                    echo "something went wrong";
```
Comment (olleharstedt) [inline application/commands/ImportSurveyCommand.php:74]: Bit imprecise :) Also should use gT for translation.

Comment (lajosarpad) [reply application/commands/ImportSurveyCommand.php:74]: Applied the change.

Comment (olleharstedt) [reply application/commands/ImportSurveyCommand.php:74]: Where? ^^ No push?

Comment (lajosarpad) [reply application/commands/ImportSurveyCommand.php]: Forgot to push it. It's pushed now. Excuse me for the inconvenience

File: application/commands/ImportSurveyCommand.php [hunk 2]
```diff
@@ -40,22 +48,33 @@ protected function importFile($filename, $furtherParams)
...
+                if (!preg_match('/^[a-zA-Z0-9_\. ]*$/', $filename)) {
+                    throw new Exception(gT("Your filename can only contain letters, digits and dot"));
                 }
                 $furtherParams = $sArgument[2] ?? null;
-                $this->importFile($filename, $furtherParams);
+                $result = $this->importFile($path, $filename, $furtherParams);
+                if (is_array($result) && isset($result['newsid'])) {
+                    echo $result['newsid'];
+                } else {
+                    echo gT("something went wrong");
```
[the remark this answers is not in the document]
Comment (lajosarpad) [reply application/commands/ImportSurveyCommand.php]: In this case the problem is that the import failed and we did not get a proper newsid in the response

[the remark this answers is not in the document]
Comment (olleharstedt) [reply application/commands/ImportSurveyCommand.php]: I think the `importSurvey` function returns some array with potential errors, but that's not really in our current must-have bucket, so let's just skip it for now.

**Pull request merged**